### ISSUE TITLE
releaseNotesFile processed incorrectly in gradle app distribution plugin

### ISSUE THREAD

**Issue by damon-beaven** [firebase/firebase-android-sdk#3720]
### [READ] Step 1: Are you in the right place?
Yes

The plugin `firebase-appdistribution-gradle:3.0.1` seems to have a problem processing `releaseNotesFile`. There could be other issues with other plugin vars but this is the one I can distinctly verify and document.

### [REQUIRED] Step 2: Describe your environment

  * Android Studio version: v4.2.2
  * Firebase Component: firebase-appdistribution-gradle
  * Component version: 3.0.1

### [REQUIRED] Step 3: Describe the problem
`firebase-appdistribution-gradle:3.0.1` is what seems to break my automated builds that upload to Firebase. I've verified that v3.0.0 worked just fine. However, 3.0.1 seems to be interpreting the vars passed to the plugin in an odd way. We generally use `System.env` to pass in vars but I removed that and hardcoded a path to `releaseNotesFile` and still received a failure to read that file from the plugin. The failure is because the plugin has prepended a path to the path I included in the plugin vars resulting in an invalid filepath.

#### Steps to reproduce:
upgrade to `firebase-appdistribution-gradle:3.0.1` from `3.0.0` in `build.gradle`

Utilize the plugin in `app/build.gradle`
```
        prototype {
            minSdkVersion 24
            targetSdkVersion 31
            applicationId '<your android applicationId>'
            firebaseAppDistribution {
                appId="<your firebase appId>"
                releaseNotesFile="C:\\jenkins\\workspace\\Android-PR/release_notes.txt"
                groups="test-automation"
            }
        }
```

Trigger a build on Jenkins. Our build system is a Window box. The build will fail at the Firebase upload step with the message below:
```
14:45:32 * What went wrong:
14:45:32 Execution failed for task ':app:appDistributionUploadPrototypeRelease'.
14:45:32 > Could not read content from C:\jenkins\workspace\Android-PR\Haiku/C:\jenkins\workspace\Android-PR/release_notes.txt
```

You can see the path I declared through the plugin is very different from the one the plugin ultimately used.

**Comment by argzdev** [comment]
Thanks for reporting, @damon-beaven. I'm unable to reproduce this issue on a Mac machine locally (without Jenkins), and I haven't tested this on a Windows machine. But I'm guessing the issue has nothing to do with whether using a Windows or Mac machine, but rather due to the integration with Jenkins.

But just to confirm, have you tried uploading without using Jenkins using a Windows machine?

Also any chance you could share with us a minimal reproducible example of this issue? And steps on how you've integrated Jenkins so we can reproduce this issue on our side? It'll greatly help us in our investigation. Thanks!

**Comment by damon-beaven** [comment]
> I was able to test this on a Windows machine locally without integrating Jenkins and I've experienced the same issue. I'll notify an engineer about this and see what we can do here. Thanks!

Thanks for the update @argzdev !

**Comment by tagboola** [comment]
Hi @damon-beaven. We shipped version `3.0.2` which fixes this issue. Please reach out it if you run into any issues after upgrading!

**Comment by damon-beaven** [comment]
@tagboola I was finally able to get back to this task and test your release. In my limited testing, this seems to resolve the issue on my end. Thanks so much!
